src/main/kotlin/io/provenance/client/grpc/PbClient.kt

```diff
@@ -77,6 +80,12 @@ open class PbClient(
     val upgradeClient = cosmos.upgrade.v1beta1.QueryGrpc.newBlockingStub(channel)
     val wasmClient = cosmwasm.wasm.v1.QueryGrpc.newBlockingStub(channel)
 
+    // @TestnetFeaturePreview
```

Comment:
@mtps did you mean to leave this commented out?

Reply:
I removed it in the spirit of quick iteration. I like the concept of guarding in this manner though.